src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,58 @@
+package calculator;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+    private static final Pattern DELIMITER_PATTERN = Pattern.compile("//(.)\\n(.*)");
+    private static final String DEFAULT_DELIMITER = "[,:]";
+    private static final int DEFAULT_RETURN_VALUE = 0;
+
```

Comment:
[단순 질문]
다른 분들 코드를 보니, 아래 와 같이 Default 생성자를 넣어두는 경우가 있었습니다. 
Default 생성자가 자동 생성 되므로 생략 가능하다고는 하지만, 명시적으로 넣어주는 게 좋을까요?

```suggestion
public class Calculator {
    Calculator() {
    }
}
```

Reply:
기본 생성자의 접근 제어자를 변경하고 싶은 경우가 아니라면, 굳이 기본 생성자를 추가할 필요는 없다고 생각합니다. :)